clients/src/main/java/org/apache/kafka/common/metrics/stats/Meter.java

```diff
@@ -77,6 +77,8 @@ public Meter(TimeUnit unit, SampledStat rateStat, MetricName rateMetricName, Met
     @Override
     public void record(MetricConfig config, double value, long timeMs) {
         rate.record(config, value, timeMs);
-        total.record(config, value, timeMs);
+        // Total metrics with Count stat should record 1.0 (as recorded in the count)
```

Comment:
This comment doesn't seem to say anything different than the code, right? The thing that wasn't clear to me is why we need to special case the total, but not the rate.

Reply:
@ijuma Thanks for the review. What I was trying to say in the comment, but probably not very clearly was that we record `1.0` here for the total because the corresponding `Count` recorded `1.0`. `Rate` with `Count` as the underlying stat counts `1.0`, so that works fine.